### PR TITLE
Save Corelli calibration manifest file functionality

### PR DESCRIPTION
**Pull request by williamfgc**
**Description of work.**
Scripts to save a manifest csv file for a list of banks with timestamp
Appends if file exists
Stores one calibration file per day
Added Tests

**Report to:** @jmborr @peterfpeterson 

**To test:**
Added unit tests on CI

Fixes #xxxx. 
*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/corelli/calibration/database.py [hunk 1]
```diff
@@ -59,6 +59,35 @@ def filename_bank_table( bankID:int, database_path:str, table_type:str = 'calibr
...
+    Function that saves or updates an existing manifest_corelli_date.csv file.
+    There is one file stored for each calibration day. If file exist it will append two columns:
+    bankID, timestamp (ISO format)
+    :param bankIDs input of bankIDs that have been calibrated
+    :param database_path location of the corelli database (absolute or relative)
+           Example: database/corelli/ for manifest file:
+                    database/corelli/manifest_corelli_YYYYMMDD.csv
+    """
+
+    time:str = datetime.now().strftime('%Y%m%d') # format YYYYMMDD
```
Comment (jmborr) [inline scripts/corelli/calibration/database.py:73]: I think this is not the correct day-stamp that should be assigned to the input bank ID's. In general, each bank calibration file that ends up being retrieved when compiling the full-instrument calibrated file has its own day-stamp. The day-stamp for a particular bank should be the day-stamp of the events file that was used to calibrate that bank.

**Review by jmborr — CHANGES_REQUESTED** [review]

File: scripts/corelli/calibration/database.py (hunk 1 above)
Comment (williamfgc) [reply scripts/corelli/calibration/database.py:73]: Sounds good, thanks @jmborr . I will accommodate and allow for an input date on the save functions.

Comment (jmborr) [reply scripts/corelli/calibration/database.py:73]: As I understand your code, if you're passing an array of bank ID's to `save_manifest_file()`, you could also pass an array of same length containing the day-stamp for each bank ID.

**Comment by williamfgc** [comment]
Superseeded by #29717

**Pull request closed without merging**